hg-fast-export.py

```diff
@@ -55,6 +55,10 @@ def wr(msg=b''):
   stdout_buffer.write(b'\n')
   #map(lambda x: sys.stderr.write('\t[%s]\n' % x),msg.split('\n'))
 
+def wr_data(data):
+  wr(b'data %d' % (len(data)))
+  wr_no_nl(data)
```

Comment:
I like `wr_data()`, but i don't like dropping the optional `\n`, as the git fast-import` manpage says:

_The LF after <raw> is optional (it used to be required) but recommended. Always including it makes debugging a fast-import stream easier as the next command always starts in column 0 of the next line, even if <raw> did not end with an LF._

Reply:
Fair enough. The good thing about the helper is that it's updated everywhere at once.